doc/build_instructions/windows_msvc.md

```diff
@@ -54,6 +56,18 @@ _Note:_ Also ensure that `python` and `python3` both point to the correct and th
  _Note:_ If you are planning to build the 64-bit version of openage, you are going to need 64-bit libraries.
  Add command line option `--triplet x64-windows` to the above command or add the environment variable `VCPKG_DEFAULT_TRIPLET=x64-windows` to build x64 libraries. [See here](https://github.com/Microsoft/vcpkg/issues/1254)
 
+  _Note:_ If you have error with it like `CMake Error at scripts/cmake/vcpkg_acquire_msys.cmake:229 (file): file failed to extract:` or a disc in exFAT you can download with this command in powershell
+  ```powershell
+  mkdir download
+  cd download
+  $zipfile = "openage-dep-x64-windows.zip"
+  Invoke-WebRequest https://github.com/SFTtech/openage-dependencies/releases/download/v0.5.0/openage-dep-x64-windows.zip -OutFile $zipfile
+  Expand-Archive -Path $zipfile -DestinationPath . -Force
+  Remove-Item $zipfile
+  (Get-ChildItem . -Recurse -File).FullName
+  ```
+  and then add `-DCMAKE_TOOLCHAIN_FILE=<pathToDownloadDir>` to the cmake configure command.
+
```

Comment:
This should be in a "Troubleshooting" section further below since this is a workaround that is not part of the build process.

Reply:
like you said below I will make a dir windows with different file to each case for windows

Reply:
case I mean section